### PR TITLE
Volume/Volume Mounts for Policy Webhook

### PR DESCRIPTION
**Pull request by anderssonw**
## Description of the change

Adds volume and volume mounts to the policy webhook deployment in the `policy-controller` chart. Uses the same structure for volume and volume mounts as the webhook deployment.

## Existing or Associated Issue(s)

No related issues.

## Additional Information

No idea how the JSON Schema is generated. Also some stuff was added into the README which I did not change. Presumably it was outdated.

## Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [x] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [ ] JSON Schema generated.
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

**Comment by cpanato** [comment]
thanks for your PR @anderssonw 
can you please squash your commits and sign the DCO?

File: charts/policy-controller/README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ The Helm chart for Policy  Controller
 | cosign.webhookName | string | `"policy.sigstore.dev"` |  |
 | imagePullSecrets | list | `[]` |  |
 | installCRDs | bool | `true` |  |
+| policywebhook.configData | object | `{}` |  |
```
Comment (hectorj2f) [inline charts/policy-controller/README.md:30]: let keep the old description ` Set the data of the `policy-config-controller` configmap`.

**Review by hectorj2f — COMMENTED** [review]
Let's keep the description to the field `configData`, otherwise it LGTM

**Review by cpanato — CHANGES_REQUESTED** [review]
+1 on the hectors comments

**Review by hectorj2f — APPROVED** [review]

**Pull request merged**